tests/testthat/transformer_grouping/scope_spaces-out.R

```diff
@@ -0,0 +1,20 @@
+# not adding line-break
+if (x) {1 + 1 + +1} else {3}
+
+# not removing line-break
+test_that("x",
+  {
+    my_test(call)
+})
+
+# do not replace assignment
+a = 3
+data_frame(a = 3)
+
+# do not resolve semicolon
+a <- function(x) x + 1;b;c
```

Comment:
I wouldn't mind a rule that converts semicolons (is there a rule in the style guide?), but this seems sufficiently low priority.